content/posts/hiero-mentorship-2026.md

```
@@ -0,0 +1,58 @@
+- [LFX Mentorship Overview](https://docs.linuxfoundation.org/lfx/mentorship)
+- [LFX Mentorship Guide for Mentees](https://docs.linuxfoundation.org/lfx/mentorship/mentees)
+
+These resources explain how to create your mentee profile, review program requirements, and apply for a mentorship program of your interest.
+
+## 🗓️ Join the Hiero Mentorship Call
+
+We also invite the community to join the **Hiero Mentorship Call**, held every **Monday at 8:00 AM PST**.
+
+This meeting is open to mentors and mentees and is a space to bring questions, discuss ideas, and get support related to the mentorship program process.
+
+You can access the calendar [here](https://zoom-lfx.platform.linuxfoundation.org/meetings/hiero?view=week)
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)